src/main/java/TTT/databaseUtils/CustomUserDAO.java

```diff
@@ -41,10 +55,74 @@ public CustomUser findCustomUser(String email) {
             userQuery.select(root).where(cb.equal(root.get("email"), email));
             CustomUser results = session.createQuery(userQuery).getSingleResultOrNull();
             return results;
-        } catch (NoResultException e) {
-            System.out.println("there is no Customer User with that email");
-            return null;
+        } catch (PersistenceException | IllegalArgumentException e) {
+            System.out.println("No entity found with email: " + email);
         }
+        return null;
     }
 
-}
+    public boolean deleteCustomUser(String email) {
+
+        if (email == null) {
+            return false;
+        }
+        email = email.trim();
+        if (email.isEmpty()) {
+            return false;
+        }
+
+        Session session = null;
+        try {
```

Comment:
Session can be created in a try with resources

Reply:
Done.